### PR TITLE
CATROID-308 remove permissions.txt

### PR DESCRIPTION
**Pull request by SchmidtChristian**
- remove Function in standalone gradle which removes values from permission file
- change Test in XstreamSerializerTest which uses the file
- add function in ProjectManager to remove Permissionfile if upgraded to higher language version
- add new if in language Version check
- increase language Version
- remove creation of permissionFile in XstreamSerializer

### CODE REVIEW

**Comment by 84n4n4** [comment]
ah, this is what this was good for...

maybe we should discuss with @wslany if we should fix this instead of removing.

in short:
this removes unused permissions from apk generator created apks. and was meant to remove eg. bluetooth or camera permissions if they are not required for the project.

**Comment by thmq** [comment]
> ah, this is what this was good for...
> 
> maybe we should discuss with @wslany if we should fix this instead of removing.
> 
> in short:
> this removes unused permissions from apk generator created apks. and was meant to remove eg. bluetooth or camera permissions if they are not required for the project.

I knew there was going to be some black magic that relied on this.

File: catroid/src/main/java/org/catrobat/catroid/ProjectManager.java [hunk 1]
```diff
@@ -441,6 +445,13 @@ private static void updateScriptsToTreeStructure(Project project) {
 		}
 	}
 
+	private static void removePermissionsFile(Project project) {
+		File permissionsFile = new File(project.getDirectory(), "permissions.txt");
```
Comment (84n4n4) [inline catroid/src/main/java/org/catrobat/catroid/ProjectManager.java]: please put "permissions.txt" somewhere in constants

**Comment by 84n4n4** [comment]
i clarified with @wslany, were going to remove this!
so this PR is fine, if you could just move the "permissions.txt" string somewhere in constants and use it from there.

thank you!

**Comment by wslany** [comment]
Thanks!!

**Pull request merged**